### PR TITLE
soft reverts #8128

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

it turns out that changing a thing that was meant to stop powergaming and unrealistic medical...caused powergaming and unrealistic medical

## How This Contributes To The Skyrat Roleplay Experience

well 8128 sure as hell didn't, this adds the area thing back, but doesn't remove the morphine thing

## Changelog

:cl:
fix: be sad, people who are terrible at medical, you now have to use surgery rooms again
/:cl:

### CODE REVIEW

File: code/modules/surgery/surgery_step.dm [hunk 1]
```diff
@@ -90,8 +90,9 @@
 
 	// Skyrat Edit Addition - reward for doing surgery in surgery
 	if(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat == DEAD)
-		modded_time *= SURGERY_SPEEDUP_AREA
-		to_chat(user, span_notice("You are able to work faster due to the patient's calm attitude!"))
+		if(is_type_in_list(get_area(target), list(/area/medical/surgery, /area/science/robotics)) && (TRAIT_FASTMED in user.status_traits) || (TRAIT_QUICK_CARRY in user.status_traits))
```
Comment (GoldenAlpharex) [inline code/modules/surgery/surgery_step.dm:93]: At a quick glance, this logic doesn'tseem to work, if you have the quick carry trait, you'll be able to do surgery mega fast on people everywhere.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/modules/surgery/surgery_step.dm (hunk 1 above)
Comment (thestubborn) [reply code/modules/surgery/surgery_step.dm:93]: i made this super super fast, if you have any idea how to actually make this work, go wild

**Comment by Iamgoofball** [comment]
:-1: I don't approve of this PR in it's current state. I can, however, propose an alternative PR.

**Comment by WitherVictor** [comment]
uh That's better than before I think. #8128 make it more hard to boost speed of surgery.Skyrat already lack of ways to boost surgery, but find a balance in the speed bonus they gives will be a hard work I guess

**Comment by GoldenAlpharex** [comment]
Considering we have #8791, this is no longer necessary.

**Pull request closed without merging**